packages/datetime/src/_timepicker.scss

```diff
@@ -98,13 +98,17 @@ $timepicker-control-width: $pt-grid-size * 3.3 !default;
 
     .#{$ns}-timepicker-input,
     .#{$ns}-timepicker-divider-text {
-      color: $input-color-disabled;
+      // disabled state always overrides over styles
+      /* stylelint-disable-next-line declaration-no-important */
+      color: $input-color-disabled !important;
```

Comment:
don't use `!important`. instead, add style rules inside the `.#{$ns}-dark` block to achieve higher specificity. this is also using the wrong color. there is a different color for disabled input text in dark theme.

you should add near the bottom of this file:

```scss
.#{$ns}-dark .#{$ns}-timepicker {
  // ...

  &.#{$ns}-disabled {
    .#{$ns}-timepicker-input,
    .#{$ns}-timepicker-divider-text,
    .#{$ns}-timepicker-arrow-button,
    .#{$ns}-timepicker-arrow-button:hover {
      color: $dark-input-color-disabled;
    }
  }
}
```

Reply:
done and tested!